sky/clouds/oci.py

```diff
@@ -231,15 +231,22 @@ def make_deploy_resources_variables(
 
         global _tenancy_prefix
         if _tenancy_prefix is None:
-            identity_client = oci_adaptor.get_identity_client(
-                region=region.name, profile=oci_conf.get_profile())
-
-            ad_list = identity_client.list_availability_domains(
-                compartment_id=oci_adaptor.get_oci_config(
-                    profile=oci_conf.get_profile())['tenancy']).data
-
-            first_ad = ad_list[0]
-            _tenancy_prefix = str(first_ad.name).split(':')[0]
+            try:
+                identity_client = oci_adaptor.get_identity_client(
+                    region=region.name, profile=oci_conf.get_profile())
+
+                ad_list = identity_client.list_availability_domains(
+                    compartment_id=oci_adaptor.get_oci_config(
+                        profile=oci_conf.get_profile())['tenancy']).data
+
+                first_ad = ad_list[0]
+                _tenancy_prefix = str(first_ad.name).split(':')[0]
+            except (oci_adaptor.get_oci().exceptions.ConfigFileNotFound,
+                    oci_adaptor.get_oci().exceptions.InvalidConfig) as e:
+                # This should only happen in testing where oci config is
+                # monkeypatched.
```

Comment:
ditto. It worth explaining why these errors will not happen in non-test cases.

Reply:
Because in non-testing env. The sky check even cannot passed.

Reply:
Let me add more comments